pkgs/development/python-modules/fpdf2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fonttools
+}:
+
+buildPythonPackage rec {
+  pname = "fpdf2";
+  version = "2.7.6";
+  format = "setuptools";
```

Comment:
Could you please use `pyproject = true` instead?
If we set `format = "setuptools"`, the builder uses a legacy installer.